content/contributing/tutorial-structure.md

```diff
@@ -0,0 +1,58 @@
+---
+html: tutorial-structure.html
+parent: contribute.html
+---
+
+# Tutorial Structure
+
+Each new XRPLF tutorial follows the same basic format.
+
+## Description
```

Comment:
This is the first of two `## Description` headers. I think this one may be better off deleted entirely (especially if you also follow my suggestion regarding moving the images under the subheaders below).

Reply:
Typo. Removed.